code/modules/loadout/categories/accessories.dm

```diff
@@ -0,0 +1,95 @@
+// --- Loadout item datums for accessories ---
+
+/// Accessory Items (Moves overrided items to backpack)
+/datum/loadout_category/accessories
+	category_name = "Accessory"
+	category_ui_icon = FA_ICON_VEST
+	type_to_generate = /datum/loadout_item/accessory
+	tab_order = 4
+
+/datum/loadout_item/accessory
+	abstract_type = /datum/loadout_item/accessory
+	/// Can we adjust this accessory to be above or below suits?
+	VAR_FINAL/can_be_layer_adjusted = FALSE
+
+/datum/loadout_item/accessory/New()
+	. = ..()
+	if(ispath(item_path, /obj/item/clothing/accessory))
+		can_be_layer_adjusted = TRUE
+
+/datum/loadout_item/accessory/get_ui_buttons()
+	if(!can_be_layer_adjusted)
+		return ..()
+
+	var/list/buttons = ..()
+
+	UNTYPED_LIST_ADD(buttons, list(
+		"label" = "Layer",
+		"act_key" = "set_layer",
+		"active_key" = INFO_LAYER,
+		"active_text" = "Above Suit",
+		"inactive_text" = "Below Suit",
+	))
+
+	return buttons
+
+/datum/loadout_item/accessory/handle_loadout_action(datum/preference_middleware/loadout/manager, mob/user, action, params)
+	if(action == "set_layer")
+		return set_accessory_layer(manager, user)
+
+	return ..()
+
+/datum/loadout_item/accessory/proc/set_accessory_layer(datum/preference_middleware/loadout/manager, mob/user)
+	if(!can_be_layer_adjusted)
+		return FALSE
+
+	var/list/loadout = manager.preferences.read_preference(/datum/preference/loadout)
+	if(!loadout?[item_path])
+		return FALSE
+
+	if(isnull(loadout[item_path][INFO_LAYER]))
+		loadout[item_path][INFO_LAYER] = FALSE
+
+	loadout[item_path][INFO_LAYER] = !loadout[item_path][INFO_LAYER]
+	manager.preferences.update_preference(GLOB.preference_entries[/datum/preference/loadout], loadout)
+	return TRUE // Update UI
+
+/datum/loadout_item/accessory/insert_path_into_outfit(datum/outfit/outfit, mob/living/carbon/human/equipper, visuals_only = FALSE)
+	if(outfit.accessory)
+		LAZYADD(outfit.backpack_contents, outfit.accessory)
+	outfit.accessory = item_path
+
+/datum/loadout_item/accessory/on_equip_item(
+	obj/item/clothing/accessory/equipped_item,
+	datum/preferences/preference_source,
+	list/preference_list = preference_source?.read_preference(/datum/preference/loadout),
```

Comment:
Default values having actual behavior is spooky and I remember it causing problems from time to time, let's avoid this